articles/sentinel/sap/sap-solution-security-content.md

```diff
@@ -87,6 +87,11 @@ The following tables list the built-in [analytics rules](deploy-sap-security-con
 | **SAP - Dynamic RFC Destination** | Identifies the execution of RFC using dynamic destinations. <br><br>**Sub-use case**: [Attempts to bypass SAP security mechanisms](#built-in-sap-analytics-rules-for-attempts-to-bypass-sap-security-mechanisms)| Execute an ABAP report that uses dynamic destinations (cl_dynamic_destination). For example, DEMO_RFC_DYNAMIC_DEST.   <br><br>**Data sources**: SAPcon - Audit Log | Collection, Exfiltration |
 | **SAP - Sensitive Tables Direct Access By Dialog Logon** | Identifies generic table access via dialog sign-in. | Open table contents using `SE11`/`SE16`/`SE16N`. <br><br>**Data sources**: SAPcon - Audit Log | Discovery |
+| **SAP - (Preview) File Downloaded From a Malicious IP Address** | User has downloaded a file from an SAP system using an IP address known to be malicious. Malicious IP addresses are obtained from [threat intelligence services](https://learn.microsoft.com/en-us/azure/sentinel/understand-threat-intelligence). | User downloaded a file from a malicious IP. <br><br>**Data sources**: SAP security Audit log, Threat Intelligence | Exfiltration |
+| **SAP - (Preview) Data exported from a production system using a transport** | Transports are like pull requests and are normally created in a dev system. This alert rule triggers incidents with medium severity when a transport is released from a prod system, bearing data taken from any table, and will create a high severity incident when the export includes data from a sensitive table.  | Transport is released from a prod system. <br><br>**Data sources**: SAP CR log, [SAP - Sensitive Tables](#tables) | Exfiltration |
+| **SAP - (Preview) Sensitive data saved into a USB drive** | SAP data exported via files and saved into a recently mounted USB drive in proximity to an execution of a sensitive transaction, a sensitive program or direct access to a sensitive table. | SAP data exported via files and saved into a USB drive. <br><br>**Data sources**: SAP Security Audit Log, DeviceFileEvents (Microsoft Defender for Endpoint), [SAP - Sensitive Tables](#tables), [SAP - Sensitive Transactions](#transactions), [SAP - Sensitive Programs](#programs) | Exfiltration |
+| **SAP - (Preview) Printing of potentially sensitive data** | User has requested printing or has printed data which is potentially sensitive. Data is considered sensitive if it is obtained by using a sensitive transaction, by executing a sensitive program or by directly accessing a sensitive table.  | User has requested printing or has printed sensitive data. <br><br>**Data sources**:  SAP Security Audit Log, SAP Spool logs, [SAP - Sensitive Tables](#tables), [SAP - Sensitive Programs](#programs) | Exfiltration |
```

Comment:
`Printing of Potentially Sensitive data'

I tried to make the phrasing more active, please check that it's still correct.

`Identifies a request or actual printing of potentially sensitive data. Data is considered sensitive if the user obtains the data as part of a sensitive transaction, execution of a sensitive program, or direct access to a sensitive table.`